.travis.yml

```diff
@@ -4,9 +4,10 @@ before_script:
  - chmod +x build.sh
 
 script: 
-  - ./build.sh RunTests
+  - ./build.sh GenerateDocs
```

Comment:
Now that I think about it, the target should be "CI". That's the one AppVeyor uses.